Integrations/integration-SCADAfence_CNM.yml

```diff
@@ -267,18 +272,85 @@ script:
         if asset_details:
             api_suffix = "/asset/traffic"
             return call_api('GET', api_suffix, asset_details)
-        raise "Invalid call for asset map (missing parameters)"
+        raise Exception("Invalid call for asset map (missing parameters)")
```

Comment:
Please use our `return_error` function instead of raise: `return_error('Invalid call for asset map (missing parameters)'`